src/App.js

```diff
@@ -1,15 +1,42 @@
-import React from "react";
+import React, { useState, useEffect} from "react";
+import axios from 'axios';
+import Title from "./Components/Title";
+import Information from "./Components/Information";
+import Video from "./Components/Video";
+
 import "./App.css";
 
-function App() {
+const App = () => {
+  const [state, setState] = useState({
+    
+    date: '',
+    explanation: "",
+    media_type: "",
+    service_version: "",
+    title:"",
+    url:""
+  })
+
+  useEffect( () => {
+    axios
+    .get('https://api.nasa.gov/planetary/apod?api_key=qyd35keQiktzxrfaHvbGgfuR9fdC1yNH49lr5QLs')
+    .then( res => {
+      console.log( "My data", res.data)
+      setState(res.data)
+    })
+    .catch(err => console.log("You know you messed up, right?", err))
+  }, [])
+
   return (
     <div className="App">
-      <p>
-        Read through the instructions in the README.md file to build your NASA
-        app! Have fun 🚀!
-      </p>
+        <Title  title={state.title} />
+        <div className="pic-of-the-day">
+          <Information date={state.date} service={state.service_version} media={state.media_type}  />
```

Comment:
You don't have to pass these separately, you can actually pass them as `state` and then call them accordingly in your child component